openmdao/docs/features/core_features/index.rst

```diff
@@ -1,3 +1,5 @@
+.. _feature_building_blocks:
```

Comment:
Your fails on Travis are coming from this.  This label is defined in another place, as noted in the error:
```Warning, treated as error:
/home/travis/build/OpenMDAO/OpenMDAO/openmdao/docs/features/core_features/index.rst:5:duplicate label feature_building_blocks, other instance in /home/travis/build/OpenMDAO/OpenMDAO/openmdao/docs/features/building_blocks/index.rst
make: *** [buildalltravis] Error 2```

So you just need to figure out what you're trying to do with this label. Can't label two things with the same name.

Reply:
fixed